compiler/rustc_middle/src/mir/mod.rs

```diff
@@ -2450,6 +2456,10 @@ impl<'tcx> Debug for Rvalue<'tcx> {
                     }),
                 }
             }
+
+            ShallowInitBox(ref place, ref ty) => {
+                write!(fmt, "ShallowInitBox({:?}, {:?})", place, ty)
```

Comment:
This looks a bit odd in the MIR dumps (with a value and a type being passed the same way), but it does match how `NullaryOp` is printed, so I guess this can be improved later on.